codalab/worker/worker_run_state.py

```diff
@@ -521,7 +520,8 @@ def finalize_run(self, run_state):
         """
         Prepare the finalize message to be sent with the next checkin
         """
-        if not run_state.failure_message and run_state.is_killed:
+        if run_state.is_killed:
+            # Set the failure message as the kill_message, which contains useful information on why a run was killed.
```

Comment:
If `failure_message` exists, we should concatenate the kill message to the failure message so we don't lose information.

Reply:
@percyliang That's a good idea. Are you okay with something like this:

/bin/bash: line 1: 7 Killed ( python test.py ) > stdout 2> stderr . Memory limit 1g exceeded.